fullstack-cert/python-projects/binary-search-workshop/main.py

```diff
@@ -0,0 +1,31 @@
+import math
+
+
+def binary_search(search_list, value):
+    path_to_target = []
+    low = 0
+    high = len(search_list) - 1
+
+    while low <= high:
+        mid = math.floor((low + high) / 2)
```

Comment:
```suggestion
        mid = (low + high) // 2
```
There's floor division operator, which makes relying on `math` module not necessary.